src/stability_sdk/client.py

```diff
@@ -246,6 +251,11 @@ def generate(
         :param samples: Number of samples to generate.
         :param safety: DEPRECATED/UNUSED - Cannot be disabled.
         :param classifiers: DEPRECATED/UNUSED - Has no effect on image generation.
+        :param guidance_preset: Guidance preset to use. See generation.GuidancePreset for supported values.
+        :param guidance_cuts: Number of cuts to use for guidance.
+        :param guidance_strength: Strength of the guidance.
```

Comment:
Default this to `0.25`, and specify/check the range? Or at least note that the default is `0.25` if not set on the backend.